ocs_ci/ocs/ui/page_objects/s3_vector_tab.py

```
@@ -0,0 +1,443 @@
+
+    def navigate_to_vector_bucket(self, bucket_name: str):
+        """
+        Click on a vector bucket name to open its detail page.
+
+        Args:
+            bucket_name (str): OBC metadata name. The S3 Vector tab displays the
+                NooBaa internal bucket name, which shares the first 26 chars with
+                the OBC name (obc-type-{first_12_hex_of_uuid}), so we search by
+                that common prefix.
+        """
+        logger.info(f"Navigating to vector bucket: {bucket_name}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)